debezium-sink/src/main/java/io/debezium/bindings/kafka/KafkaDebeziumSinkRecord.java

```diff
@@ -47,8 +80,17 @@ public long offset() {
     }
 
     @Override
-    public List<String> keyFieldNames() {
-        throw new RuntimeException("Not implemented");
+    public Struct kafkaCoordinates() {
+        return kafkaCoordinates;
+    }
+
+    @Override
+    public Struct kafkaHeader() {
+        return kafkaHeader;
+    }
+
+    public Map<String, FieldDescriptor> allFields() {
```

Comment:
```suggestion
    @Override
    public Map<String, FieldDescriptor> allFields() {
```

Reply:
Done and also other `@Override` annotations added where they were missing.